sw/device/silicon_creator/manuf/lib/flash_info_fields.c

```diff
@@ -104,6 +104,13 @@ const flash_info_field_t kFlashInfoFieldTpmCikAttestationKeySeed = {
     .byte_offset = kFlashInfoFieldTpmCikKeySeedIdx * kAttestationSeedBytes,
 };
 
+const flash_info_field_t kFlashInfoFieldDevSeedSeed = {
+    .partition = 0,
+    .bank = 0,
+    .page = 4,
+    .byte_offset = kFlashInfoFieldDevSeedSeedIdx * kAttestationSeedBytes,
+};
+
```

Comment:
does this belong in closed source custom extension?

Reply:
I am not sure about this one.

Should   
```
kFlashInfoFieldTpmCekKeySeedIdx = 4,
kFlashInfoFieldTpmCikKeySeedIdx = 5,
``
and their corresponding entries in the flash info table also move into extension?

Reply:
Yeah I think so. since they are specific to this SKU. You can do this in a follow up PR if you would like though.

Reply:
moved constants to ext. specific tree